build/development-guide/smart-contracts/get-started-evm/use-hardhat.md

```diff
@@ -5,15 +5,13 @@ This example is inpired by the [Getting Started](https://hardhat.org/getting-sta
 ## Prerequisites
 - node
 - yarn
-- npx
-
 
 ## Let's create a basic sample project
 
 ```bash=
 mkdir hardhat && cd hardhat
 yarn add hardhat
-npx hardhat
+node ./node_modules/hardhat/internal/cli/cli.js
```

Comment:
we could just do `yarn hardhat` which is a shortcut for `node ./node_modules/hardhat/internal/cli/cli.js`, but either way is fine